dashboard/app/controllers/learning_goal_evaluations_controller.rb

```diff
@@ -0,0 +1,33 @@
+class LearningGoalEvaluationsController < ApplicationController
+  def create
+    learning_goal_evaluation = LearningGoalEvaluation.new(learning_goal_evaluation_params)
+
+    learning_goal_evaluation.save!
+
+    render json: learning_goal_evaluation
+  end
+
+  def update
+    params.require(:id)
+    learning_goal_evaluation = LearningGoalEvaluation.find_by_id(params[:id])
+    if learning_goal_evaluation
```

Comment:
should we return a 404 if a learning goal evaluation doesn't exist here?

Reply:
Added an error message. Do I need to add an additional unit test for this?